core/templates/core/user-voted-content.html

```diff
@@ -90,7 +91,11 @@ <h3 class="h3 mx-5">{{ post.title }}</h3>
         </div>
         {% empty %}
         <div class="d-flex justify-content-center mt-5">
+            {% if url == 'user_upvoted' %}
```

Comment:
This could only change the word upvoted/downvoted not the whole line. This change will improve maintenance, but is not critical.